elasticsearch/mapping/index_settings.json

```diff
@@ -74,7 +76,16 @@
                     }
                 },
                 "misc": {
-                    "type": "object"
```

Comment:
There were no obvious consequences. This is how it is currently set up and Anthony reported that everything worked as expected. This did prompt me to have a good look and i found an extra { which would have caused problems so good that is sorted.